File: eco-core/core-backend/src/main/java/com/willfp/eco/internal/config/yaml/EcoYamlConfigSection.java
package com.willfp.eco.internal.config.yaml;

import org.bukkit.configuration.ConfigurationSection;
import org.jetbrains.annotations.NotNull;

public class EcoYamlConfigSection extends EcoYamlConfigWrapper<ConfigurationSection> {
    public EcoYamlConfigSection(@NotNull final ConfigurationSection section) {
        this.init(section);
    }
}

File: eco-core/core-backend/src/main/kotlin/com/willfp/eco/internal/config/yaml/EcoYamlConfigSection.kt
package com.willfp.eco.internal.config.yaml

import com.willfp.eco.core.placeholder.StaticPlaceholder
import org.bukkit.configuration.ConfigurationSection

class EcoYamlConfigSection(section: ConfigurationSection, injections: Collection<StaticPlaceholder> = emptyList()) : EcoYamlConfigWrapper<ConfigurationSection>() {
    init {
        init(section)
        this.injections = injections.toMutableList()
    }
}